cupy_xarray/tests/test_accessors.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
 import pytest
 import xarray as xr
-from xarray.core.pycompat import dask_array_type
+from xarray.namedarray.pycompat import DuckArrayModule
 
 import cupy_xarray  # noqa: F401
 
+dsk = DuckArrayModule("dask")
+dask_array_type = dsk.type
```

Comment:
Let's just not do that 

```suggestion
try:
    import dask.array
    dask_array_type = dask.array.Array
    
 else:
     dask_array_type = None
```

There is some new `importlib` way of doing this that I can't remember now

Reply:
Ok, done at 6229f999808e7b6826b0f7092ffa85f159548d2f.

> There is some new `importlib` way of doing this that I can't remember now

Do you mean [`importlib.util.find_spec`](https://docs.python.org/3/library/importlib.html#importlib.util.find_spec)? That would allow us to get to the `dask.array` module, but not the `dask.array.Array` type. Unless there's another function I'm missing.

Reply:
yeah that one but maybe it's not a big deal here/